.drone/drone.yml

```diff
@@ -251,6 +251,17 @@ steps:
     ops_tools_fn_upload:
       from_secret: ops_tools_fn_upload
 
+- name: deploy-tempo-serverless-lambda
+  image: amazon/aws-cli
+  commands:
+  - cd ./cmd/tempo-serverless/lambda
+  - aws s3 cp tempo-serverless*.zip s3://dev-us-east-0-tempo-dev-02-fn-source
+  environment:
+    AWS_ACCESS_KEY_ID-tempo-dev-02:
```

Comment:
will this work? doesn't this just need to be `AWS_ACCESS_KEY_ID` ?

Reply:
Ah yes, thank you.

Reply:
Updated with some help from the standard lib.